crates/sui-replay/src/replay.rs

```diff
@@ -458,8 +573,8 @@ impl LocalExec {
                 "Genesis replay not supported: {}, skipping transaction",
                 tx_digest
             );
-            return Ok(tx_info.effects);
-            // return Err(LocalExecError::GenesisReplayNotSupported { digest: *tx_digest });
+            //return Ok(tx_info.effects);
```

Comment:
Which one should it be?

Reply:
Very good callout. I want to revert it to a warning but still trying to figure out what to return for `ExecutionSandboxState` for genesis. For now keeping it an error is simple, but will have a way forward by next PR